### PR TITLE
chore: show trusted issuer name alongside issuer domain

### PR DESCRIPTION
**Pull request by HJunyuan**
**What does this PR do?**
- Allow trusted issuer to have their name shown alongside issuer domain
- Trusted TLDs: `*.gov.sg` and `*.edu.sg`
- Example:
  ```text
  Certificate issued by
  NTU.EDU.SG
  NANYANG TECHNOLOGICAL UNIVERSITY
  ```

| Before | After |
|---|---|
| Screenshot 2022-03-07 at 2 54 06 PM | Screenshot 2022-03-07 at 2 54 15 PM |

### CODE REVIEW

File: src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx [hunk 1]
```diff
@@ -35,27 +37,62 @@ export const getV2IdentityVerificationText = (
...
+          if (id.from === "registry") return <li {...props}>{id.name}</li>;
+          else if (isTrusted(id.location))
+            return (
+              <li {...props}>
+                {id.location}
+                <br />
+                {id.name}
+              </li>
+            );
+          else return <li {...props}>{id.location}</li>;
```
Comment (simboonlong) [inline src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx:87]: are the tests breaking for u?

Screenshot 2022-03-09 at 3 40 14 PM

File: src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx [hunk 2]
```diff
@@ -35,27 +37,62 @@ export const getV2IdentityVerificationText = (
       const indexedDnsDidFragment = Array.isArray(dnsDidFragment?.data) ? dnsDidFragment?.data[index] : undefined;
 
       if (OpencertsRegistryVerificationValidData.guard(indexedRegistryFragment))
-        return { location: indexedRegistryFragment.name.toUpperCase(), from: "registry" };
+        return {
+          name: indexedRegistryFragment.name.toUpperCase(),
+          location: indexedRegistryFragment.value,
+          from: "registry",
+        };
```
Comment (simboonlong) [inline src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx:44]: able to point me to various fixtures to test out?

File: src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx (hunk 1 above)
Comment (HJunyuan) [reply src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx:86]: Whoops, missed that out. Not sure why [CircleCi workflow](https://app.circleci.com/pipelines/github/OpenCerts/opencerts-website) is not shown in the below checks 🤔

In any case, I've updated the tests already.

File: src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx (hunk 2 above)
Comment (HJunyuan) [reply src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx:42]: If you would like to test locally, can specify the `TRUSTED_TLDS` environment variable or just override it here:

https://github.com/OpenCerts/opencerts-website/blob/c60fa6d80c8f7b83822406529fd4d287645d3b2a/next.config.js#L28

Can just set it to `openattestation.com` and use the OpenCerts DID example found in the [gallery](https://action.openattestation.com/?q=%7B%22type%22%3A%22DOCUMENT%22%2C%22payload%22%3A%7B%22uri%22%3A%22https%3A%2F%2Fgallery.openattestation.com%2Fstatic%2Fdocuments%2Ftranscript-did.opencert%22%2C%22permittedActions%22%3A%5B%22STORE%22%5D%2C%22redirect%22%3A%22https%3A%2F%2Fdev.opencerts.io%22%7D%7D).

Comment (HJunyuan) [reply src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx:42]: I've also set the TRUSTED_TLDS environment variable in `dev.opencerts.io` to:
```text
gov.sg,edu.sg,openattestation.com
```

You can try it out using the deploy preview here: https://deploy-preview-667--opencerts-development.netlify.app/?q={"type":"DOCUMENT","payload":{"uri":"https://gallery.openattestation.com/static/documents/transcript-did.opencert","permittedActions":["STORE"],"redirect":"https://dev.opencerts.io"}}

**Review by simboonlong — APPROVED** [review]
lgtm

**Pull request merged**